lib/salus/repo_searcher.rb

```diff
@@ -157,9 +157,8 @@ def run_rg(*args)
       data = IO.popen(args, chdir: @path_to_repo).read
       return [] if data == ""
 
-      files = data.lines.map(&:strip)
+      data.lines.map(&:strip)
       # files are all relative to @path_to_repo
```

Comment:
Can we remove the comment because we are not using **files** variable anymore?
# files are all relative to @path_to_repo